tools/c7n_tencentcloud/c7n_tencentcloud/resources/mysql.py

```diff
@@ -0,0 +1,44 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n_tencentcloud.provider import resources
+from c7n_tencentcloud.query import ResourceTypeInfo, QueryResourceManager
+from c7n_tencentcloud.utils import PageMethod, isoformat_datetime_str
+import pytz
+
+
+@resources.register("mysql")
+class MySQL(QueryResourceManager):
+    """
+    mysql: distributed data storage service, relational databases
+    https://www.tencentcloud.com/document/product/236/5147
+    """
+
+    class resource_type(ResourceTypeInfo):
+        """resource_type"""
+        id = "InstanceId"
+        endpoint = "cdb.tencentcloudapi.com"
+        service = "cdb"
+        version = "2017-03-20"
+        enum_spec = ("DescribeDBInstances", "Response.Items[]", {})
+        metrics_instance_id_name = "InstanceId"
+        paging_def = {"method": PageMethod.Offset, "limit": {"key": "Limit", "value": 20}}
+        resource_prefix = "instanceId"
+        taggable = True
+
+        datetime_fields_format = {
+            "CreateTime": ("%Y-%m-%d %H:%M:%S", pytz.timezone("Asia/Shanghai"))
+        }
+
+    def augment(self, resources):
+        for resource in resources:
+            cli = self.get_client()
+            resp = cli.execute_query("DescribeDBInstanceInfo",
```

Comment:
can we move this to an encryption filter so we don't need to fetch it unless its being used for a policy assertion.

also generally its preferable when adding additional attributes to use a c7n: prefix annotation to distinguish from native ones. we generally make exceptions for tags, but beyond that its generally a legacy motion.

Reply:
Hi kapilt, use c7n: prefix is ok.
Do you have any definition for native attributes and additional attributes?
In our mindset, those directly from API are native, those generated by code are additional.

Reply:
Yes, that's accurate, thanks

Reply:
According to this rule, I think there are many codes need to be modified.

Reply:
Added encryption filter